docs/install/tkgs.rst

```diff
@@ -302,3 +319,16 @@ When you try to create a Notebook Server, you may meet the following error:
     FailedCreate 1s (x2 over 1s) statefulset-controller create Pod test-01-0 in StatefulSet test-01 failed error: pods “test-01-0” is forbidden: PodSecurityPolicy: unable to admit pod: []
 
 This error occurs because Notebook Server creation needs pod creation, and you did not configure the pod security policy correctly. To solve this error, you need to configure pod security policy based on :ref:`configure pod security policy`.
+
+cert-manager webhook is not ready
+---------------------------------
+
+cert-manager is used by Kubeflow components to provide certificates for admission webhooks. When you try to install Kubeflow, you may meet the following error about cert-manager:
```

Comment:
``cert-manager``? (Or The "cert-manager" component? This "cert-manager" should either be a "word, with first letter capitalized, or ``xx``, or "xx")